src/features/logout/index.ts

```diff
@@ -0,0 +1,3 @@
+import LogoutRowButton from './ui/LogoutRowButton';
```

Comment:
Let's instead write`export * from './ui'` for the index.ts files. That will make it easier to add further changes